src/catch2/catch_tostring.hpp

```diff
@@ -41,6 +41,13 @@ namespace Catch {
 
     namespace Detail {
 
+        inline std::size_t catch_strnlen(const char *str, std::size_t n) {
+            auto ret = std::memchr(str, '\0', n);
```

Comment:
Better to use [std::char_traits<char>::find()](https://en.cppreference.com/w/cpp/string/char_traits/find). That has proper types and not `void*`. it's also `constexpr` if needed.

Reply:
Done. It is actually the nicer solution.